optaplanner-distribution-internal/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
shouldn't it be located in  `optaplanner-distribution/internal` or `optaplanner-distribution/optaplanner-distribution-internal`? :thinking:  I mean, in case additional distributions in the future...

Reply:
I don't think there will be more distributions. This is a partial distribution that is only used to build the complete  `optaplanner-distribution` that has moved to optaplanner-quickstarts.